R/transformed.R

```diff
@@ -37,14 +37,43 @@ format.dist_transformed <- function(x, ...){
   )
 }
 
+#' @export
+support.dist_transformed <- function(x, ...) {
+  support <- support(x[["dist"]])
+  lim <- field(support, "lim")[[1]]
+  lim <- SW(x[['transform']](lim))
+  if (all(!is.na(lim))) {
+    lim <- sort(lim)
+  }
+  field(support, "lim")[[1]] <- lim
+  support
+}
+
 #' @export
 density.dist_transformed <- function(x, at, ...){
-  density(x[["dist"]], x[["inverse"]](at))*abs(vapply(at, numDeriv::jacobian, numeric(1L), func = x[["inverse"]]))
+  inv <- function(v) SW(x[["inverse"]](v))
+  jacobian <- vapply(at, numDeriv::jacobian, numeric(1L), func = inv)
+  d <- density(x[["dist"]], inv(at)) * abs(jacobian)
+  limits <- field(support(x), "lim")[[1]]
+  closed <- field(support(x), "closed")[[1]]
+  if (!any(is.na(limits))) {
+    `%op1%` <- if (closed[1]) `<` else `<=`
+    `%op2%` <- if (closed[2]) `>` else `>=`
+    d[which(at %op1% limits[1] | at %op2% limits[2])] <- 0
```

Comment:
Much neater, looks great! :tada:
Minor suggestion, naming `%op1%` and `%op2%` to be more informative would improve readability.

e.g. `%lt%` and `%gt%` or some other clearer name.